### ISSUE TITLE
Remove ensure from tests

### ISSUE THREAD

**Issue by edigiacomo** [ARPA-SIMC/arkimet#178]
From `ensure(statement)` to e.g. `wassert(actual(actual_result) == expected_result)`

**Comment by spanezz** [comment]
The use of `ensure` is now (as of branch issue177) limited to tests that include the `arki/tests/legacy.h` header, which is currently 10 sources

**Comment by spanezz** [comment]
Correction, currently 0 sources :)

This will be fixed once `issue177` is merged into master
